tmt/steps/provision/connect.py

```diff
@@ -110,8 +111,31 @@ def go(self):
             data['key'] = key
 
         # And finally create the guest
-        self._guest = tmt.Guest(data, name=self.name, parent=self.step)
+        self._guest = GuestConnect(data, name=self.name, parent=self.step)
 
     def guest(self):
         """ Return the provisioned guest """
         return self._guest
+
+
+class GuestConnect(tmt.Guest):
+    """
+    Guest connection class.
+    """
+
+    def reboot(self, hard=False, machine_name=None):
+        super.reboot(hard, name)
+        conn_timeout = 60 * 4
```

Comment:
From my point of view it would be nicer to have this defined at the top of the module with capital letters since this is a constant. That would be more consistent with the rest of the modules

Reply:
Solved.